pkg/image/apiserver/registry/imagestream/strategy.go

```diff
@@ -463,6 +463,16 @@ func (v *TagVerifier) Verify(old, stream *imageapi.ImageStream, user user.Info)
 	if old != nil && old.Spec.Tags != nil {
 		oldTags = old.Spec.Tags
 	}
+
+	type streamId struct {
```

Comment:
we generally don't put these within functions, just as private types in the package